app/controllers/state_file/questions/pending_federal_return_controller.rb

```diff
@@ -0,0 +1,19 @@
+module StateFile
+  module Questions
+    class PendingFederalReturnController < QuestionsController
+      def self.show?(_intake)
+        false
```

Comment:
Is this false because this page will be directly linked on the IRS side?

Reply:
yes! i considered not putting it in the flow at all but in the end it was nice that the routes just come with the navigation and also it sort of demonstrates "where" clients would encounter this page